### ISSUE TITLE
[svelte-check] TypeError: Cannot read property 'Glob' of undefined

### ISSUE THREAD

**Issue by metonym** [sveltejs/language-tools#1089]
**Describe the bug**

After upgrading from svelte-check v2.2.0 to v2.2.1, I receive the following error when running `yarn svelte-check`:

```sh
$ svelte-check --workspace=test
/Users/<redacted>/<redacted>/<redacted>/node_modules/svelte-check/dist/src/index.js:1372
glob_1.Glob;
       ^

TypeError: Cannot read property 'Glob' of undefined
    at Object.<anonymous> (/Users/<redacted>/<redacted>/<redacted>/node_modules/svelte-check/dist/src/index.js:1372:8)
    at Module._compile (internal/modules/cjs/loader.js:1063:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1092:10)
    at Module.load (internal/modules/cjs/loader.js:928:32)
    at Function.Module._load (internal/modules/cjs/loader.js:769:14)
    at Module.require (internal/modules/cjs/loader.js:952:19)
    at require (internal/modules/cjs/helpers.js:88:18)
    at Object.<anonymous> (/Users/<redacted>/<redacted>/<redacted>/node_modules/svelte-check/bin/svelte-check:2:1)
    at Module._compile (internal/modules/cjs/loader.js:1063:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1092:10)
error Command failed with exit code 1.
```

**To Reproduce**

My `package.json` includes `svelte-check` and `svelte` as development dependencies.

The `test` folder contains a single Svelte file that tests TypeScript definitions for the component.

The following commands throw the aforementioned error:

- `yarn svelte-check`
- `yarn svelte-check --workspace=test`

**Expected behavior**

**Screenshots**

**System (please complete the following information):**
 - OS: MacOS
 - IDE: VSCode
 - Plugin/Package: `svelte-check`

**Comment by ccadieux** [comment]
I am getting this using node 16.4.2.

**Comment by dummdidumm** [comment]
Fixed with 2.2.2
There appears to be a bug in a recent Rollup version which caused this when bundling the package.

**Comment by metonym** [comment]
Thank you – can confirm it works in 2.2.2
